### PR TITLE
AD guidance on multiple forests and domain syncs

### PR DESCRIPTION
**Pull request by johnthompson365**
This is a starter for 10 for potential clearer guidance on supported AD topologies. I have provided the main different options I can think of. Be good to work through each to give a description and guidance for each.

### CODE REVIEW

File: source/deployment/sso-ldap.md [hunk 1]
```diff
@@ -58,17 +58,40 @@ Note: Make sure that at least 1 LDAP user is in Mattermost or the sync will not
...
+
+Description
+Configuration example
+
+_Multi-forest without trusts_
+
+We do not support this topology.
+
+_Considerations_
+Searching using GC as opposed to LDAP
```
Comment (tejaycar) [inline source/deployment/sso-ldap.md]: What is GC?

**Comment by wiersgallak** [comment]
@johnthompson365 Do you have a customer that you are planning on working with or were you looking to get some support to help go through these? Or yourself?  The format looks good to me.

**Comment by johnthompson365** [comment]
No specific customer, just responding to a number of queries raised recently on the subject. It would be great to get some support from yourself/dev to work through what we currently are able to do in the product and what we can't. Some of ti may just be on the wording but some may require testing. I can dedicate some time to spin up some of the AD topologies in a lab if needed.

**Comment by hanzei** [comment]
Hey @johnthompson365,

What are the next steps on this PR? Does it need work or review? Just want to make sure it doesn't get forgotten.

**Comment by wiersgallak** [comment]
Per conversation with @johnthompson365 closing this since it has gone stale and other research work was conducted and can be documented

**Pull request closed without merging**